### DISCUSSION TITLE
How to set parameters specifically  in ParaView?

### DISCUSSION THREAD

**Discussion by Devin-one** [PMEAL/OpenPNM#2346] [Q&A]
Hello Professor, This tutorial ,https://github.com/PMEAL/OpenPNM/blob/dev/examples/notebooks/io/view_vtp_in_paraview.md , omits many processes, which is a little unfriendly to us beginners. The following picture is the Unsatisfactory result I got.Can you elaborate on how to set parameters in detail? It's best to have an example. Thank you very much !
![3](https://user-images.githubusercontent.com/93040662/159169884-99106ba6-64a2-461d-92b7-95e973cee231.png)

**Comment by jgostick** [comment]
Hi @Liqiwen123 
Paraview takes a bit of getting used to.  The 'throat' visualization in particular is a pain.  I don't have time to make a detailed tutorial, but it's on my (long) todo list.  I think you should be able to find general paraview tutorials on how to color glyphs and such.  There is nothing fancy about openpnm's data.  You just need to adapt the settings until you get what you want.
